openssl-sys/src/evp.rs

```diff
@@ -405,6 +405,18 @@ extern "C" {
     pub fn EVP_PKEY_keygen(ctx: *mut EVP_PKEY_CTX, key: *mut *mut EVP_PKEY) -> c_int;
 }
 
+cfg_if! {
+    if #[cfg(any(ossl110, libressl280))] {
```

Comment:
I'm not sure about the libressl version here, but all builds succeed.

Reply:
Looking at the libressl release tags, 2.8.0 seems to be correct.